msm/mycroft_skills_manager.py

```diff
@@ -117,9 +117,7 @@ def list_defaults(self):
 
         if self.platform not in skill_groups:
             LOG.error('Unknown platform:' + self.platform)
-        return chain(*[
-            skill_groups.get(i, []) for i in {'default', self.platform}
-        ])
+        return chain(skill_groups.get(self.platform, []))
```

Comment:
`chain` isn't needed anymore and we might as well fall back to the default skills via `.get(self.platform, skill_groups.get('default', []))`.

Reply:
Ok, I think I've updated accordingly. A non-existing platform now defaults to 'defaults' if I understand things correctly and a list is returned instead of a chain.